.travis.yml

```diff
@@ -1,12 +1,14 @@
-# Documentation: http://docs.travis-ci.com/user/languages/julia/
 language: julia
 os:
   - linux
-#  - osx # disabled until queues are reasonable
+  - osx
 julia:
   - 0.6
   - 0.7
-#  - nightly
+  - 1.0
+  - nightly
```

Comment:
Please drop nightly. It slows CI times for all JuliaOpt packages because of the shared queues.